src/pages/index.js

```diff
@@ -14,19 +16,41 @@ import {
   profileFormDescription,
   editForm,
   addForm,
+  confirmDeleteForm,
   newProfileFormName,
   newProfileFormDescription,
+  profilePicture,
 } from "../utils/utils.js";
 import "./index.css";
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c6945e47-b548-4e1c-b20e-926f9841fa5f",
+    "Content-Type": "application/json",
+  },
+});
+
 const editPopup = new PopupWithForm(".edit-popup", changeProfileInfo);
 const addPopup = new PopupWithForm(".add-popup", addLocationCard);
+const profilePicturePopup = new PopupWithForm(
+  ".profile-picture-popup",
+  changeProfilePicture
+);
 const editFormValidator = new FormValidator(options, editForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```